dvc/external_repo.py

```diff
@@ -68,32 +70,49 @@ def __init__(self, root_dir, url):
         self._set_upstream()
 
     def pull_to(self, path, to_info):
-        try:
-            out = None
-            with suppress(OutputNotFoundError):
-                out = self.find_out_by_relpath(path)
+        """
+        Pull the corresponding file or directory specified by `path` and
+        checkout it into `to_info`.
+
+        It works with files tracked by Git and DVC, and also local files
+        outside the repository.
+        """
+        out = None
+        path_info = PathInfo(self.root_dir) / path
+
+        with suppress(OutputNotFoundError):
+            (out,) = self.find_outs_by_path(path_info, strict=False)
```

Comment:
```suggestion
            (out,) = self.find_outs_by_path(fspath(path_info), strict=False)
```
3.5 will fail with PathInfo

Reply:
thanks, @pared :+1: